bazelrc/.bazelrc.test.0.25.0

```diff
@@ -0,0 +1,46 @@
+# Copyright 2016 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This .bazelrc file contains a specific set of flags 
+# to test incompatible Bazel flags for Bazel 0.25.0
+# Only for testing. Do not use these flags for your RBE configuration!
+
+build:remote --jobs=50
+build:remote --spawn_strategy=remote
+build:remote --strategy=Javac=remote
+build:remote --strategy=Closure=remote
+build:remote --strategy=Genrule=remote
+build:remote --define=EXECUTOR=remote
+build:remote --remote_cache=remotebuildexecution.googleapis.com
+build:remote --remote_executor=remotebuildexecution.googleapis.com
+build:remote --tls_enabled=true
+build:remote --remote_timeout=3600
+build:remote --auth_enabled=true
+
+# Turn on incompatible_enable_cc_toolchain_resolution and don't
+# use crosstool_top
+build:remote --incompatible_enable_cc_toolchain_resolution
```

Comment:
Why not use the existing bazelrc and use this flag in the test?

Reply:
The default bazelrc flag has action_env=BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN=1 which, as I found out doing this test, is incompatible with use of incompatible_enable_cc_toolchain_resolution.

Also, I want to define a way that we can do tests for different versions of Bazel with different incompatible  flags, changing only the minimum set of artifacts for a new Bazel release (i.e., not just for this specific incompatible_enable_cc_toolchain_resolution flag). This way all you will need to add for a new version of bazel (and new set of incompatible flags) is the bazelrc file and the new trigger.

Reply:
Moved all files to yaml file and removed custom bazelrc file